### ISSUE TITLE
Don't generate stats.json in production

### ISSUE THREAD

**Issue by jonsherrard** [shortlist-digital/tapestry-wp#187]
Stats.json for reasonable sized applications are already at 6-7mb, on smaller hosts like heroku review apps, this is probably taking a fair amount of CPU and memory to generate.

Dynamically including:
```
new StatsPlugin('../.tapestry/stats.json', {
  chunkModules: true
}),
```
Might help things out... worth a test though.

Thoughts welcome @mathewhawley @thomasdigby

**Comment by mathewhawley** [comment]
Sounds like a good idea to me! 👍 Could add whatever flag variable we are using for the `chunkModules` value: `chunkModules: DEBUG`?

**Comment by jonsherrard** [comment]
Should we default to no stats.json output at all, unless a dev asks for it with a flag?

**Comment by thomasdigby** [comment]
Flags might be a no-go with Commander. I don't see the issue with creating it in development mode?

**Comment by thomasdigby** [comment]
Closed by #189
